flocker/ca/_script.py

```diff
@@ -16,10 +16,74 @@
 from ..common.script import (flocker_standard_options, ICommandLineScript,
                              FlockerScriptRunner)
 
-from ._ca import (CertificateAuthority, CertificateAlreadyExistsError,
-                  KeyAlreadyExistsError)
+from ._ca import (RootCredential, ControlCredential,
+                  CertificateAlreadyExistsError, KeyAlreadyExistsError,
+                  PathError)
 
 
+@flocker_standard_options
```

Comment:
Don't think you want the standard options on sub-commands, seems like it should only be on the main `Options` class.